examples/app-settings/src/settings.ts

```diff
@@ -48,15 +46,6 @@ export const settings = defineSettings({
         },
     ],
     example2: [
-        {
-            type: 'input',
-            id: 'input-password',
-            inputType: 'password',
-            label: 'Your password',
```

Comment:
Same for the password  inputType, to hide the input gives the impression that it then cannot be visible anymore, but nothing prevents the App from just getting this values like any other.